service/src/main/java/bio/terra/externalcreds/services/FenceProviderService.java

```diff
@@ -117,8 +113,12 @@ public LinkedAccount createLink(
     }
   }
 
-  public void deleteFenceLink(String userId, Provider provider) {
-    bondService.deleteBondLinkedAccount(userId, provider);
+  public void deleteBondFenceLink(String userId, Provider provider) {
+    try {
+      bondService.deleteBondLinkedAccount(userId, provider);
+    } catch (Exception ex) {
```

Comment:
What do you think about only logging this warning for non-404 errors?

Reply:
I don't think its too big of a deal to just log all errors. It would be good to have logs for how many times bond was called to delete a non-existent account. Theoretically, we should see warnings for that increase as time goes on and more accounts exist in ECM only. It would be a good metric to measure ECM's uptake!